packages/react-error-overlay/src/containers/StackFrame.js

```diff
@@ -78,16 +75,10 @@ class StackFrame extends Component {
       return;
     }
     const {
-      _originalFileName: sourceFileName,
-      _originalLineNumber: sourceLineNumber,
+      _originalFileName: fileName,
+      _originalLineNumber: lineNumber,
     } = this.props.frame;
-    // Keep this in sync with react-error-overlay/middleware.js
```

Comment:
This comment was lost when the code was moved.

Reply:
Added an updated comment.